src/v2/Apps/ViewingRoom/Components/ViewingRoomHeader.tsx

```diff
@@ -46,14 +51,13 @@ export const ViewingRoomHeaderFragmentContainer = createFragmentContainer(
  */
 const ViewingRoomHeaderLarge: React.FC<ViewingRoomHeaderProps> = props => {
   const {
-    viewingRoom: { heroImageURL, title },
+    viewingRoom: { image, title },
   } = props
 
-  // FIXME: Wire up. Currently Gemini is returning a 500 from this, not sure why.
-  // const resizedHeroImageURL = resize(heroImageURL, {
-  //   width: 600,
-  //   convert_to: "jpg",
-  // })
+  const heroImageURL = resize(image?.imageURLs?.normalized, {
+    width: 1200,
+    convert_to: "jpg",
```

Comment:
Do we need to convert to jpg here if the source images are already in jpg format?

Reply:
I kept it from previous implementation. Though probably not needed if we allow only `.jpg` from initial upload.